db-schema/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 /// Column families alias type
 pub type Col = &'static str;
 /// Total column number
-pub const COLUMNS: u32 = 16;
+pub const COLUMNS: u32 = 17;
```

Comment:
@zhangsoledad @yangby-cryptape Do we need migration to add a column now?

Reply:
> Do we need migration to add a column now?

Required.

> Or what happens if I open the database with 17 columns in the ckb version which knows only 16 columns?

Panic.

Ref: https://github.com/nervosnetwork/ckb/commit/f782409b7fb22c9d378200540ec5bae8a0f12858

Reply:
so i need add migration part, right?  and migration still need to done with hardfork? 
please tell me more about migration.

Reply:
https://github.com/nervosnetwork/ckb/pull/2878

Reply:
already add  db migration commit